src/script/sign.cpp

```diff
@@ -9,6 +9,7 @@
 #include <policy/policy.h>
 #include <primitives/transaction.h>
 #include <script/standard.h>
+#include <script/signingprovider.h>
```

Comment:
Nit: `i` comes before `t`

(I am leaving the ones so far as is, as you seem to have paid some attention to ordering of includes, but I am not bringing this up in future occurrences during this review.)